README.md

```diff
@@ -1,21 +1,61 @@
 Zinc
 ====
 
-Zinc is an incremental compilation module of sbt with the sbt-agnostic API.
+Zinc is the incremental compiler for Scala. Most Scala developers use it daily
+without noticing -- it's embedded in key build tools like sbt, CBT and pants.
```

Comment:
Pants is using it, and I know that CBT is interested in using it as well. Bazel will not.